vue/components/organisms/pickers/pickers.vue

```diff
@@ -16,6 +16,18 @@
                 v-on:click="slideLeftParts"
             />
             <ul class="parts-container" ref="partsPicker">
+                <li
+                    class="part button button-part button-extra"
+                    v-for="button in beforeButtonsParts"
+                    v-bind:key="button.id"
+                    v-on:click="onButtonPartClick(button.event)"

```

Comment:
When forwarding events, always include the native event (`$event`).
```suggestion
                    v-on:click="onButtonPartClick($event, button.event)"

```